### PR TITLE
e2e: Automatically inject a newline escape character in the Logf function

### PR DESCRIPTION
**Pull request by timflannagan**
Update the Logf helper function which wraps the GinkgoWriter() utility and automatically inject a newline escape character when the input string doesn't already contain that.

### CODE REVIEW

File: test/e2e/plain_provisioner_test.go [hunk 1]
```diff
@@ -30,6 +30,9 @@ const (
 )
 
 func Logf(f string, v ...interface{}) {
+	if !strings.HasSuffix(f, "\n") {
+		f += "\n"
+	}
 	fmt.Fprintf(GinkgoWriter, f, v...)
```
[the remark this answers is not in the document]
Comment (timflannagan) [reply test/e2e/plain_provisioner_test.go:36]: Yeah I'm not too worried about that edge case tbh but I can update to your suggestion.

**Review by tylerslaton — APPROVED** [review]

**Review by akihikokuroda — APPROVED** [review]
Looks good to me.  Thanks!  

Is this `Logf` used anywhere now?

**Comment by timflannagan** [comment]
> Is this Logf used anywhere now?

I don't think so but I'd image we'll need to start using this more as debugging e2e failures is already a challenge (and I thought we had an issue around this I guess we never created that).
